newdle/templates/deletion_email.txt

```diff
@@ -1,8 +1,15 @@
-Hi,
+Hi {{ participant_name }},
 
 {{ creator }} has deleted the newdle "{{ title }}".
 
 {%- if comment %}
 
-Comment: {{ comment }}
+Comment:
+{{ comment }}
+
 {%- endif -%}
+
+Click on the button below for more details.
+{{ summary_link }} 
+
+This email was sent by {{ summary_link }}
```

Comment:
I don't think this looks good. I would use the plaintext version for an email signature, and just "newdle" instead of a link.

```

--
This email was sent by newdle
```

PS: Feel free to just leave the .txt emails as they are in maste - the issue is mainly about the HTML mails anyway :)

Reply:
I change the .txt emails for the (almost) originals just adding the greetings